src/main/java/com/example/a1project/Task.css

```diff
@@ -0,0 +1,72 @@
+.root {
+	-fx-background-color: #ffffff;
+	-fx-background-radius: 25;
+	-fx-border-color: #77B3D4;
+	-fx-border-width: 6 6 6 6;
+	-fx-border-radius: 20;
+}
+
+#addTaskButton { 
+	-fx-background-color: #77B3D4;
+	-fx-background-radius: 20;
+	-fx-font-family: "Avenir Next";
+	-fx-font-size: 48;
+	-fx-font-weight: bold; 
+}
```

Comment:
I think it would be good to include a constant for our most used colours so that we can change it easily if need be. The hex code has been included quite a few times here so would be hard to update.

Reply:
Looks perfect!